contrib_bots/test-bots

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import sys
+import unittest
+
+from unittest.mock import MagicMock, patch
+from unittest import TestCase
+
+from run import get_lib_module
+from bot_lib import BotHandlerApi, StateHandler
+
+class BotTestCase(TestCase):
+
+    def mock_test(self, messages, message_handler):
+        # type: (List[Dict[str, str]], Function) -> None
+        # Mocking BotHandlerApi
+        with patch('__main__.BotHandlerApi') as MockClass:
+            instance = MockClass.return_value
+
+            for message in messages:
+                # Send message to the concerned bot
+                message_handler.handle_message(message, MockClass(), StateHandler())
+
+                # Check if BotHandlerApi is sending a reply message.
+                # This can later be modified to assert the contents of BotHandlerApi.send_message
+                assert instance.send_message.called is True
```

Comment:
Yeah! We can always do that, but the initially idea for writing these tests was to have this test-suite framework for bots independent from the rest of the code-base.

This can definitely be added later! Not a priority as of now. :)